nengo_viz/static/viz_netgraph_item.js

```diff
@@ -36,7 +36,7 @@ VIZ.NetGraphItem = function(ng, info) {
     this.minHeight = 5;
 
     this.expanded = false;
-    
+
```

Comment:
I took a stab at doing this manually the way @tbekolay showed me...  the result is in

https://github.com/nengo/nengo_viz/commits/non_default_pointers_test

It seems to work....  here are the steps I did:

- ```git checkout non_default_pointers```
- ```git rebase -i HEAD~2``` (do an interactive rebase of the last 2 commits in this repository)
- change the text ```pick``` to ```edit``` beside the commit we want to remove things from (this tells the rebase to stop at that point and let us manually do things)
- ```git reset HEAD^``` (undo the commit that we are stopped at)
- ```git add nengo_viz/components/netgraph.py``` (this file is fine)
- ```git add nengo_viz/components/pointer.py``` (so is this one)
- ```git add -p nengo_viz/static/viz_netgraph_item.js``` (we only want part of this file)
- it will ask you what to do with each part of the file.  say ```n``` (no) to all the whitespace changes and ```y``` to the actual changes
- ```git commit``` (you will have to copy in your original commit message)
- ```git checkout .``` (throw away the remaining changes)
- ```git rebase --continue``` (finish the rebase)